kong/plugins/aws-lambda/handler.lua

```diff
@@ -62,6 +65,21 @@ local function get_now()
   return ngx_now() * 1000 -- time is kept in seconds with millisecond resolution.
 end
 
+local function validate_http_status_code(status_code)
+  if not status_code then
+    return false
+  end
+  local status_code_str = tostring(status_code)
+  local m, err = re_match(status_code_str, VALID_HTTP_STATUS_CODE_REG, 'jo')
+  if err then
+    kong.log.err(err)
+    return false
+  end
+  if not m then
```

Comment:
```suggestion

  if not m then
```

Style: need a newline before and after a new code block.

Reply:
Thanks for pointing it out! Seems that I also forget to add two blank lines between functions

Fixed